### PR TITLE
Fixture/update config files

### PR DESCRIPTION
**Pull request by steven-heng**
Npm audit found 0 vulnerabilities
Modify rules command of package.json and Makefile

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,25 +1,27 @@
...
 	echo sonar.projectVersion=`cat package.json | python -c "import json,sys;obj=json.load(sys.stdin);print obj['version'];"` >> sonar-project.properties
 	wget https://sonarsource.bintray.com/Distribution/sonar-scanner-cli/sonar-scanner-2.8.zip
 	unzip sonar-scanner-2.8.zip
 ifdef CI_PULL_REQUEST
-	@sonar-scanner-2.8/bin/sonar-runner -e -Dsonar.analysis.mode=preview -Dsonar.github.pullRequest=${shell basename $(CI_PULL_REQUEST)} -Dsonar.github.repository=$(REPO_SLUG) -Dsonar.github.oauth=$(GITHUB_TOKEN) -Dsonar.login=$(SONAR_LOGIN) -Dsonar.password=$(SONAR_PASS) -Dsonar.host.url=$(SONAR_HOST_URL)
+	@sonar-scanner-2.8/bin/sonar-scanner -e -Dsonar.analysis.mode=preview -Dsonar.github.pullRequest=${shell basename $(CI_PULL_REQUEST)} -Dsonar.github.repository=$(REPO_SLUG) -Dsonar.github.oauth=$(GITHUB_TOKEN) -Dsonar.login=$(SONAR_LOGIN) -Dsonar.password=$(SONAR_PASS) -Dsonar.host.url=$(SONAR_HOST_URL)
 endif
 ifeq ($(CIRCLE_BRANCH),develop)
 	@sonar-scanner-2.8/bin/sonar-runner -e -Dsonar.analysis.mode=publish -Dsonar.host.url=$(SONAR_HOST_URL) -Dsonar.login=$(SONAR_LOGIN) -Dsonar.password=$(SONAR_PASS)
```
Comment (jkernech) [inline Makefile]: Use `sonar-scanner` here as well, `sonar-runner` is deprecated

**Review by jkernech — CHANGES_REQUESTED** [review]

**Review by mrister — DISMISSED** [review]
tests are failing

**Review by jkernech — CHANGES_REQUESTED** [review]
Tests need to be fixed

**Comment by mrister** [comment]
Fixed this but I was not able to fix sonar with ciircleCi2 so I left it commented (some permission error on exec).

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  build:
+    docker:
+    - image: dialonce/nodejs:lts-carbon # the primary container, where your job's commands are run
+    steps:
+    - checkout # check out the code in the project directory
+    - run: apk add --no-cache make python
+    - run: make deps
+    - run: make test
+## Comment out one permission denied is fixed
+#    - run: make sonar
```
Comment (Spring3) [inline .circleci/config.yml:12]: I think you are lacking env variables here. That's why it was failing with some permission errors

**Review by Spring3 — APPROVED** [review]

File: .circleci/config.yml (hunk 2 above)
Comment (mrister) [reply .circleci/config.yml:12]: they are set up in the ENV section on circle ci page

Comment (mrister) [reply .circleci/config.yml:12]: as the ones for SWS s3 which work

**Review by Spring3 — CHANGES_REQUESTED** [review]
Could you also please include the .circleci folder to the .npmignore?

**Comment by mrister** [comment]
@Spring3 done!

**Review by Spring3 — APPROVED** [review]

**Pull request merged**